src/Backend/Modules/MediaLibrary/Domain/MediaItem/Type.php

```diff
@@ -61,6 +61,8 @@ public static function fromMimeType(string $mimeType): self
                 return self::fromString($mediaItemType);
             }
         }
+
+        throw new Exception('MimeType could not be resolved');
```

Comment:
Maybe use a less generic exception?

Reply:
Used InvalidArgument since it is about a string and not a specific class